### DISCUSSION TITLE
Suggestion for Korean TV EPG

### DISCUSSION THREAD

**Discussion by lmnl01** [iptv-org/community#113] [Q&A] [answered]
Hello, is it possible to use these sources for EPG?

https://m.seezntv.com/liveTV?ch_no=285&category_id=1&channel_type=epg
https://www.wavve.com/live/index.html

**Comment by BellezaEmporium** [comment, marked as answer]
Please ask directly onto @iptv-org/epg

**Reply by lmnl01** [reply]
My bad. I'm so sorry

**Reply by BellezaEmporium** [reply]
No worries. It is just a matter of keeping things well sorted in here.
